rebalance/rebalance.py

```diff
@@ -404,9 +432,11 @@ def maybe_rebalance_pairs(plugin: Plugin, ch1, ch2, failed_pairs: list):
 
 
 def maybe_rebalance_once(plugin: Plugin, failed_pairs: list):
-    channels = get_our_channels(plugin)
+    channels = get_open_channels(plugin)
     for ch1 in channels:
         for ch2 in channels:
+            if ch1 == ch2:
+                continue
```

Comment:
No need for this, I think. If `maybe_rebalance_pairs` calculates the sending amount correctly, in this case the amount will be zero, and there will be no try to rebalance.
(Of course, this can remain here, to make the thing safer/clearer.)